### PR TITLE
Feature/hvg 631 connect payment update copy design

### PR DESCRIPTION
**Pull request by virrios**
[HVG-631] Connect Payment > Update copy and design

## What?
The changes are mostly copy and design updates.
_Figma [https://www.figma.com/file/eRR5ACHtbsPWDd4eUauWv1/Onboarding-Offer-page?node-id=255%3A4253&viewport=535%2C244%2C0.2588714361190796]()_
Refactored som style objects to styled css.
Added translation object to adyen checkout to change button name

## Why?
To make the connect payment screen up to date with the simple sign release

_Referenced ticket [https://hedvig.atlassian.net/browse/HVG-631]()_

### Screens
Norwegian (iphone simulator)
AdyenSImSm
**Norwegian (iphone 6/7/8 in devtools)**
AdyenIphoneDevSM

Swedish (iphone simulator)
swedishIphoneSm
**Swedish (iphone 6/7/8 in devtools)**
SwdishDevSm

Desktop
image

image

[HVG-631]: https://hedvig.atlassian.net/browse/HVG-631

### CODE REVIEW

File: src/client/pages/ConnectPayment/sections/ConnectPayment.tsx [hunk 1]
```diff
@@ -5,105 +5,131 @@ import { Market, useMarket } from 'components/utils/CurrentLocale'
...
+  color: ${colorsV3.white};
+  ${MEDIUM_SCREEN_MEDIA_QUERY} {
+    width: 100%;
+  }
+  ${MOBILE_SCREEN_MEDIA_QUERY} {
+    font-size: 1rem;
+  }
+`
+const InstructionWrapper = styled(ConnectText)`
+  margin-bottom: 1rem;
```
Comment (styrelius) [inline src/client/pages/ConnectPayment/sections/ConnectPayment.tsx:110]: I think it would be nice with some more `margin-bottom` here, and I think it looks like the design in Figma has that as well. But also this element is pretty redundant - the styles here could be moved to the `InstructionsList` and this could be removed? Or am I missing something?

File: src/client/pages/ConnectPayment/sections/ConnectPayment.tsx [hunk 2]
```diff
@@ -5,105 +5,131 @@ import { Market, useMarket } from 'components/utils/CurrentLocale'
...
-  },
-})
+const InstructionsList = styled.ol`
+  list-style: none;
+  padding: 0;
+  counter-reset: styled-number;
+`
+const InstructionItem = styled.li`
+  counter-increment: styled-number;
+  :: before {
```
Comment (styrelius) [inline src/client/pages/ConnectPayment/sections/ConnectPayment.tsx:129]:
```suggestion
  :before {
```

File: src/translations/en_NO.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
-  "ONBOARDING_CONNECT_DD_HEADLINE": "Connect payment",
-  "ONBOARDING_CONNECT_DD_CTA": "Connect payment",
-  "ONBOARDING_CONNECT_DD_PRE_HEADLINE": "Psst, don't forget...",
-  "ONBOARDING_CONNECT_DD_BODY": "Money will be drawn the 27th each month. We only start charging you once your insurance is activated.",
+  "ONBOARDING_CONNECT_DD_HEADLINE": "only two things left to do",
+  "ONBOARDING_CONNECT_DD_CTA": "Connect payment - activate Hedvig",
+  "ONBOARDING_CONNECT_DD_PRE_HEADLINE": "Welcome to Hedvig,",
+  "ONBOARDING_CONNECT_DD_BODY": "We are so happy to have you as a member. To start using your new home insurance, you need to:",
```
Comment (styrelius) [inline src/translations/en_NO.json:5]: Shouldn't say `home insurance`  since it might be travel insurance you've sign up for. So just `insurance`/`forsikring` is better. So it should probably be the same for all English translations. And since these should be _translations_  and nothing else I guess we should just say `försäkring`/`insurance` in Sweden as well?

File: src/translations/en_DK.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "ONBOARDING_CONNECT_DD_HEADLINE": "Connect payment",
-  "ONBOARDING_CONNECT_DD_CTA": "Connect payment",
+  "ONBOARDING_CONNECT_DD_CTA": "Connect payment - activate Hedvig",
   "ONBOARDING_CONNECT_DD_PRE_HEADLINE": "Psst, don't forget...",
   "ONBOARDING_CONNECT_DD_BODY": "Money will be drawn the 1st each month. We only start charging you once your insurance is activated.",
```
Comment (styrelius) [inline src/translations/en_DK.json:5]: So even if these are not used they should be changed to the new copy.

**Review by styrelius — COMMENTED** [review]
Nice! 🙌 
Minor thing but maybe this is a bit tight?
Screenshot 2021-01-18 at 17 22 02

Also, I assume this can't be removed? But I'm pretty sure it wasn't there before 🤔 
image

And some other comments below 👇

File: src/client/utils/mediaQueries.ts [hunk 5]
```diff
@@ -1 +1,5 @@
+export const MS_BP = 800
+export const MOBILE_BP = 450
 export const LARGE_SCREEN_MEDIA_QUERY = '@media screen and (min-width: 1000px)'
+export const MEDIUM_SCREEN_MEDIA_QUERY = `@media (max-width: ${MS_BP}px)`
+export const MOBILE_SCREEN_MEDIA_QUERY = `@media (max-width: ${MOBILE_BP}px)`
```
[the remark this answers is not in the document]
Comment (virrios) [reply src/client/utils/mediaQueries.ts:5]: Good to know, I saw some mixed usages and i based it on what was there before, but I will change to this so its inline! 👍

File: src/client/components/buttons.tsx [hunk 6]
```diff
@@ -18,7 +18,7 @@ export const Button = styled.button<ButtonProps>`
   line-height: ${(props) => (props.size === 'lg' ? `1rem` : `1.25rem`)};
   width: ${(props) => (props.fullWidth ? '100%' : 'auto')};
   padding: ${(props) =>
-    props.size === 'lg' ? `1.5rem 2.5rem` : `0.875rem 1.5rem`};
+    props.size === 'lg' ? `1.5rem 2.5rem` : `1.125rem 1.5rem`};
```
[the remark this answers is not in the document]
Comment (virrios) [reply src/client/components/buttons.tsx:21]: I thought that it was a global design update? But i can dubbel check with mange

File: src/translations/en.json [hunk 7]
```diff
@@ -1,8 +1,8 @@
 {
-  "ONBOARDING_CONNECT_DD_HEADLINE": "Connect direct debit",
-  "ONBOARDING_CONNECT_DD_CTA": "Connect direct debit",
-  "ONBOARDING_CONNECT_DD_PRE_HEADLINE": "Psst, don't forget...",
-  "ONBOARDING_CONNECT_DD_BODY": "Money will be drawn via direct debit the 27th each month. We only start charging you once your insurance is activated.",
+  "ONBOARDING_CONNECT_DD_HEADLINE": "only two things left to do",
+  "ONBOARDING_CONNECT_DD_CTA": "Connect payment - active Hedvig",
```
[the remark this answers is not in the document]
Comment (virrios) [reply src/translations/en.json:3]: 🙈

File: src/client/pages/ConnectPayment/sections/ConnectPayment.tsx (hunk 2 above)
Comment (gustaveen) [reply src/client/pages/ConnectPayment/sections/ConnectPayment.tsx:129]: For some context: https://css-tricks.com/to-double-colon-or-not-do-double-colon/
TL;DR double colon is the correct syntax for pseudo-elements, but the old syntax that I'm used to (single colon) works for both pseudo-elements and pseudo-selectors and hence you don't have to think about what is what 😄

Comment (gustaveen) [reply src/client/pages/ConnectPayment/sections/ConnectPayment.tsx:129]: Ah and the extra space as you pointed out @styrelius , didn't see that at first 👍

File: src/client/pages/ConnectPayment/sections/ConnectPayment.tsx [hunk 8]
```diff
@@ -5,105 +5,131 @@ import { Market, useMarket } from 'components/utils/CurrentLocale'
...
-  fontWeight: 400,
-})
+  ${MOBILE_SCREEN_MEDIA_QUERY} {
+    display: initial;
+    ${(props) =>
+      props.addWhiteSpace &&
+      `
+      :before {
+        content:'\\00a0';
+      }`}
```
Comment (gustaveen) [inline src/client/pages/ConnectPayment/sections/ConnectPayment.tsx:95]: This solution does the trick for sure! But I think it's easier to understand what the code does if just having a margin :) 
```suggestion
        margin-left: 0.375rem;
```

File: src/client/utils/mediaQueries.ts (hunk 5 above)
[the remark this answers is not in the document]
Comment (styrelius) [reply src/client/utils/mediaQueries.ts:5]: Yeah, it's a bit confusing with the breakpoints 😅 And we have probably not communicated properly to you the things I wrote here, sorry about that. The idea with this `mediaQueries` file was to make it less confusing and less all over the place - to align it more.

File: src/client/pages/ConnectPayment/sections/ConnectPayment.tsx [hunk 9]
```diff
@@ -5,105 +5,131 @@ import { Market, useMarket } from 'components/utils/CurrentLocale'
...
+  margin-bottom: 2.25rem;
+  font-size: 3rem;
+  color: ${colorsV3.white};
+  ${MOBILE_SCREEN_MEDIA_QUERY} {
+    text-align: center;
+    margin-bottom: 2rem;
+    font-size: 2rem;
+    line-height: 2rem;
+  }
+`
```
Comment (gustaveen) [inline src/client/pages/ConnectPayment/sections/ConnectPayment.tsx:78]: According to Figma, adjust the line-heights a bit
```suggestion
const Heading = styled.h1`
  margin-top: 0;
  margin-bottom: 2.25rem;
  font-size: 3rem;
  line-height: 3.5rem;
  color: ${colorsV3.white};
  ${MOBILE_SCREEN_MEDIA_QUERY} {
    text-align: center;
    margin-bottom: 2rem;
    font-size: 2rem;
    line-height: 2.5rem;
  }
`
```

File: src/client/pages/ConnectPayment/sections/ConnectPayment.tsx [hunk 10]
```diff
@@ -27,84 +25,87 @@ const Background = styled.div`
...
   addWhiteSpace?: boolean
 }
 const HeaderPart = styled.span<HeaderPartProps>`
-  display: block;
   font-family: ${fonts.FAVORIT};
   font-weight: 400;
+  ${(props) =>
+    props.addWhiteSpace &&
+    `
+        margin-left: 0.375rem;
```
Comment (virrios) [inline src/client/pages/ConnectPayment/sections/ConnectPayment.tsx:90]: lol, soo much smarter!

**Review by gustaveen — APPROVED** [review]
Whoop whoop, good job Victor! 🎉
